xml/System.Security.Cryptography/ECDiffieHellmanCng.xml

```diff
@@ -1043,9 +1039,22 @@
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the size, in bits, of the key modulus used by the asymmetric algorithm.</summary>
+        <value>The size, in bits, of the key modulus used by the asymmetric algorithm.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+Setting this property to its current value has no visible effect.
+Setting this property to a new legal value discards the current key, but defers creation of a new key until one is needed.
+
+Because key sizes do not uniquely identify elliptic curves, the use of the property setter is discouraged. To generate a new key on a specific curve, use the <xref:System.Security.Cryptography.ECDiffieHellmanCng.GenerateKey%2A> method.
+
+]]></format>
+        </remarks> 
+        <exception cref="T:System.Security.Cryptography.CryptographicException">
+          <paramref name="value" /> is not permitted by <see cref="P:System.Security.Cryptography.ECDiffieHellmanOpenSsl.LegalKeySizes" />.</exception>
```

Comment:
The DocId for LegalKeySizes was not found. Apparently it does not exist in the *Ssl class, but it seems to exist in the *Cng class. Is this correct?:

```suggestion
          <paramref name="value" /> is not permitted by <see cref="P:System.Security.Cryptography.ECDiffieHellmanCng.LegalKeySizes" />.</exception>
```